pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -147,16 +169,45 @@ func (m *Multicluster) AddMemberCluster(clientset kubernetes.Interface, metadata
 		if valicationWebhookController != nil {
 			go valicationWebhookController.Start(stopCh)
 		}
+		//add config controller
+		remoteConfigController, err := m.createRemoteConfigController(clusterID, config, opts)
+		if err != nil {
+			return fmt.Errorf("failed to create remote Config controller for cluster %v: %v", clusterID, err)
+		}
+		serviceEntryStore := serviceentry.NewServiceDiscovery(remoteConfigController, model.MakeIstioStore(remoteConfigController), m.XDSUpdater)
```

Comment:
do we need a store for each istio resource?

Reply:
Trying to understand if it can also handle configs beyond service entry, workload entry or k8s alpha service resources...

Reply:
This only works for service,  not for other configs, like gateway/vs/dr. etc.

Reply:
Need to update IstioConfigCache in s.Env, and it is an aggregrated cache.